pkg/alertmanager/alertmanager_metrics_test.go

```diff
@@ -1093,16 +1184,34 @@ func newNotifyMetrics(r prometheus.Registerer) *notifyMetrics {
 			Buckets:   []float64{1, 5, 10, 15, 20},
 		}, []string{"integration"}),
 	}
-	for _, integration := range integrations {
+	for _, integration := range []string{
```

Comment:
Intentionally duplicated the list here, because that's what's in prometheus's [notify.go](https://github.com/prometheus/alertmanager/blob/0f3ab929fa37ad2b7bb2ef2aae3512940308622f/notify/notify.go#L283). I didn't want to disturb the "purity" of the code that is copied from prometheus.

Reply:
These tests currently work against copy/pasted code from prometheus, which is subject to drift if prometheus changes (as it did here).

@gotjosh, WDYT about exporting some of these objects like metrics structs? Then, we could test against the actual vendored prometheus code, and avoid this kind of test drift in the future. Would alertmanager accept such a change, or is it too separated from prometheus?

Reply:
I see no harm in that if it's a straightforward change 👍